salt/common/tools/sbin/so-status

```diff
@@ -290,22 +287,32 @@ main() {
 
 # {% endraw %}
 
+is_tty() {
+    __tty=0
+    [ -t 1 ] && __tty=1
+    # don't print colors if NO_COLOR is set to anything
+    [ "${#NO_COLOR}" -ne 0 ] && __tty=0
+}
+
```

Comment:
I don't usually work with salt. How do you update the script with it?

Reply:
Definitely don't remove stuff if you don't know its purpose, that's an easy way to break things. 

To answer your question about updating a script with salt: there is a master file for every script that's located in `/opt/so/saltstack/default/salt/common/tools/sbin` which the `common` salt state copies into `/usr/sbin` after running them through a tool called jinja, which replaces variables defined using its syntax. (Check `so-import-pcap` in the master directory for a good example of what I'm talking about re: jinja syntax.) For reference, the common state can be run using the following command:
```bash
sudo salt-call state.apply common
```

That being said, since we don't currently have much in the way of developer documentation I would do some reading on saltstack and take a closer look at the rest of the project to get a better idea of how Security Onion works as a whole. This project relies fairly heavily on salt and some minimal knowledge of its major features is important to contribute code that works as expected.